src/System.Text.Json/tests/JsonPropertyTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
I don't think this file needs the Newtonsoft.Json using-imports; and probably can get rid of a lot of these... can we pare it down to the ones that matter?